SU-SWS/jumpstart_ui
# Update to Decanter version 6.0.1

# READY FOR REVIEW

# Summary
- Version constrain Decanter at version 6.x.x

# Review By (Date)
- January 16th

# Urgency
- Not urgent

# Steps to Test

1. Check out this branch
2. run `nvm use`
3. run `npm install`
4. run `npm run publish`
5. Ensure no changes to files
6. Ensure package.lock and decanter is set to version 6.0.1

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
